documentation/modules/metrics/proc-metrics-cruise-control.adoc

```diff
@@ -0,0 +1,62 @@
+// This assembly is included in the following assemblies:
+//
+// metrics/assembly-metrics.adoc
+
+[id='cruise-control-monitoring-{context}']
+= Presenting Cruise Control metrics in Grafana
+
+[role="_abstract"]
+If you are using Prometheus and Grafana to monitor built-in Kafka metrics,
```

Comment:
Do we really need this sentence in each of the procedures? Can't we just say that _You can scrape metrics from ..._

Reply:
This procedure has been removed, but I aded a line in the _Deploying Prometheus metrics configuration_ section.

> Prometheus scrapes metrics from target HTTP endpoints.